### DISCUSSION TITLE
Welcome to LawEnforcementResources Discussions!

### DISCUSSION THREAD

**Discussion by AndrewRathbun** [Digital-Forensics-Discord-Server/LawEnforcementResources#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kpatt6276** [comment]
My name is Karen Patterson, and I've been a Digital Forensic Analyst with the Ocala Police Department for just under a year.  I just came back from IACIS Mobile Device Examiner class in Orlando and my head is still swimming!  I had a question for Cellebrite, and the Solutions Engineer I spoke with recommended this Discord channel very highly.  Looking forward to lots of learning.

**Reply by AndrewRathbun** [reply]
Have you joined the server? Do you need assistance joining? Happy to have you!

**Comment by kpatt6276** [comment]
I’m not sure – lol.  I’m not familiar with Discord at all and new to forensics.  I was able to log in, but I didn’t see discussions and such.

From: Andrew Rathbun ***@***.***>
Sent: Friday, May 23, 2025 9:07 PM
To: Digital-Forensics-Discord-Server/LawEnforcementResources ***@***.***>
Cc: Karen Patterson ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [Digital-Forensics-Discord-Server/LawEnforcementResources] Welcome to LawEnforcementResources Discussions! (Discussion #7)

Have you joined the server? Do you need assistance joining? Happy to have you! — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented. Message ID: <Digital-Forensics-Discord-Server/LawEnforcementResources/repo-discussions/7/comments/13253115@ github. com>


Have you joined the server? Do you need assistance joining? Happy to have you!

**Reply by AndrewRathbun** [reply]
When you join, make sure you send an email from your professional email to digitalforensicsdiscord@gmail.com so we can validate your LE status and grant you access. Then you'll see all the channels!

**Comment by kpatt6276** [comment]
Perfect – I’ll do that now.  Thank you!


From: Andrew Rathbun ***@***.***>
Sent: Friday, May 23, 2025 9:16 PM
To: Digital-Forensics-Discord-Server/LawEnforcementResources ***@***.***>
Cc: Karen Patterson ***@***.***>; Comment ***@***.***>
Subject: [EXTERNAL] Re: [Digital-Forensics-Discord-Server/LawEnforcementResources] Welcome to LawEnforcementResources Discussions! (Discussion #7)

When you join, make sure you send an email from your professional email to digitalforensicsdiscord@ gmail. com so we can validate your LE status and grant you access. Then you'll see all the channels! — Reply to this email directly, view


When you join, make sure you send an email from your professional email to ***@***.******@***.***> so we can validate your LE status and grant you access. Then you'll see all the channels!
